src/main/java/org/cbioportal/application/file/export/services/VirtualStudyAwareExportService.java

```diff
@@ -0,0 +1,90 @@
+package org.cbioportal.application.file.export.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.io.Writer;
+import java.util.Set;
+import org.cbioportal.application.file.export.exporters.ExportDetails;
+import org.cbioportal.application.file.export.exporters.Exporter;
+import org.cbioportal.application.file.utils.FileWriterFactory;
+import org.cbioportal.legacy.service.VirtualStudyService;
+import org.cbioportal.legacy.web.parameter.VirtualStudy;
+import org.cbioportal.legacy.web.parameter.VirtualStudyData;
+import org.cbioportal.legacy.web.parameter.VirtualStudySamples;
+
+public class VirtualStudyAwareExportService implements Exporter {
```

Comment:
Better named a VirtualStudyExportDecorator

Reply:
I'll name it `VirtualStudyExportDecoratorService` just to follow naming convention of classes in the services package. Let me know what you think.